Gnosis-Builders/faucet-ui
# feat: Make Footer responsive & add Tools

## Explanation

- Update the Footer to be responsive
- Added the logo to mobile & tablet view as well
- Fixed the footer items alignments
https://trello.com/c/4Ljepe8D/20-fix-footer-responsiveness

## Manual Testing Steps

- Check the responsiveness of Footer & inform me if anything weird
- Check the links of the footer items
- Ensure proper alignment of Footer items

## Pre-merge author checklist

- [X] Sufficient automated test coverage has been added (N/A)
